### PR TITLE
Update comments and options in config files

### LINKED ISSUE #200: Update default configs
Especially as moved to AMS

### PR DESCRIPTION
**Pull request by tofu-rocketry**
Resolves #200 

- Set AMS as default messaging system.
- Add AMS hosts (both prod and devel) as options.
- Emphasise note about the server_cert.
- Add token as commented out option.
- Copy log level comments from sender to receiver conf.

### CODE REVIEW

File: conf/receiver.cfg [hunk 1]
```diff
@@ -25,17 +26,22 @@ check_crls: false
 
 [messaging]
 # If using AMS this is the project that SSM will connect to. Ignored for STOMP.
-ams_project:
+ams_project: accounting
 
 # Destination to which SSM will listen.
-destination: /queue/ssm2test
+destination: ssm2test
```
Comment (RedProkofiev) [inline conf/receiver.cfg:32]: If ssm2test is a dedicated testing architecture, then this is fine, but the intent is for this to be deliberately changed by the user.  So, 'changeme' or blank would be good.

File: conf/receiver.cfg [hunk 2]
```diff
@@ -25,17 +26,22 @@ check_crls: false
...
+#token:
 
 # Accepted messages will be written to <path>/incoming
 # Rejected messages will be written to <path>/reject
 path: /var/spool/apel
 
 [logging]
 logfile: /var/log/apel/ssmreceive.log
+# Available logging levels:
+# DEBUG, INFO, WARN, ERROR, CRITICAL
```
Comment (RedProkofiev) [inline conf/receiver.cfg:44]: SSM uses at least debug and info (remember that verbosity PR?)  Would this not disrupt that?

**Review by Sae126V — COMMENTED** [review]
Looks good to me, if we decide to provide default values

File: conf/receiver.cfg (hunk 1 above)
Comment (tofu-rocketry) [reply conf/receiver.cfg]: Done, though went with the blank option.

**Review by RedProkofiev — APPROVED** [review]
All looks fine to me.

**Pull request merged**